bitnami/harbor/Chart.yaml

```diff
@@ -34,4 +34,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-harbor-registry
   - https://github.com/bitnami/bitnami-docker-harbor-registryctl
   - https://goharbor.io/
-version: 9.1.1
+version: 9.1.2
```

Comment:
IMHO it is best if we use a minor for this, as it's adding a new feature (i.e. use secret for managing the credentials).